src/pathfinding_service/service.py

```diff
@@ -253,3 +275,13 @@ def create_token_network_for_address(
         )
         token_network_listener.start()
         self.token_network_listeners.append(token_network_listener)
+
+    def handle_message(self, message: SignedMessage):
+        if isinstance(message, UpdatePFS):
+            self.on_pfs_update(message, UpdatePFS)
+            # TODO validate message
+        else:
+            log.info('Ignoring unknown message type')
+
+    def on_pfs_update(self, message, UpdatePFS):
+        print(f"das ist die message: {message}")
```

Comment:
no german please, and better make this a log message.

Reply:
changed to English, will be deleted soon. This is just for debugging